src/Codeception/Test/Feature/CodeCoverage.php

```diff
@@ -26,8 +26,20 @@ public function codeCoverageEnd($status, $time)
             return;
         }
 
+        if (method_exists($this, 'getLinesToBeCovered')) {
```

Comment:
could you use create an interface for that. Sonething like `StrictCoverageInterface` and make Cest implementing that. This way you can simplify code by checking this interface only.